api/proto/teleport/legacy/types/types.proto

```diff
@@ -4237,6 +4237,8 @@ message ClusterAlertSpec {
     (gogoproto.stdtime) = true,
     (gogoproto.nullable) = false
   ];
+  // HostID is the id of the auth server that generated the alert.
```

Comment:
Nit: I would say something generic like "HostID is id of the node the alert applies to". It does not necessarily have to be auth server, maybe it's an alert for a particular node in the cluster.